### PR TITLE
Feature/event image upload

### PR DESCRIPTION
**Pull request by patrick-replogle**
# Description
***Please merge backend pull request #78 first since these changes rely on it***
I added the ability to upload an image in the create event form and replaced the stock image with the user's actual uploaded image in pages 3 and 4 of the form. 

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Change Status

- [x] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?
I've tested in react and pgadmin to make sure the images are being sent back and forth from the frontend and backend.

# Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

**Comment by PaulMEdwards** [comment]
[Get photo upload working on create-event forms](https://trello.com/c/GoXPCRrZ/67-get-photo-upload-working-on-create-event-forms)

File: src/components/events/create-event/FormPageOne.js [hunk 1]
```diff
@@ -246,7 +246,7 @@ const FormPageOne = ({ handleChange, values, setPage, setFieldValue }) => {
               <MenuItem value={5}>Sunday Sports</MenuItem>
               <MenuItem value={6}>Kids Play Date</MenuItem>
               <MenuItem value={7}>Puppy Play Date</MenuItem>
-              <MenuItem value={8}>Puppy Play Date</MenuItem>
+              <MenuItem value={8}>Cat Play Date</MenuItem>
```
Comment (PaulMEdwards) [inline src/components/events/create-event/FormPageOne.js:249]: 😸

File: src/components/events/create-event/FormPageThree.js [hunk 2]
```diff
@@ -27,78 +28,151 @@ const FormPageThree = ({
...
+          <div style={{ display: "flex", width: "100%", minHeight: "200px" }}>
+            <img
+              style={{
+                maxWidth: "50%",
+                border: "8px solid #58D473",
+                borderRadius: "25px",
+                maxHeight: "300px",
+              }}
+              src={photo}
+              alt="bbq"
```
Comment (PaulMEdwards) [inline src/components/events/create-event/FormPageThree.js:41]: Again, maybe use something generic like "Event Photo".

File: src/components/events/create-event/FormPageOne.js (hunk 1 above)
Comment (Aaroneld) [reply src/components/events/create-event/FormPageOne.js:249]: 🐈

**Pull request merged**